### PR TITLE
Fix Bridge tasks from resuming on new head

### PR DESCRIPTION
**Pull request by dimroc**
https://www.pivotaltracker.com/story/show/156238127

### CODE REVIEW

File: store/models/orm_test.go [hunk 1]
```diff
@@ -64,24 +64,49 @@ func TestPendingJobRuns(t *testing.T) {
...
+		expected []string
+	}{
+		{
+			"single status",
+			[]models.RunStatus{models.RunStatusPendingBridge},
+			[]string{seedIds[0]},
+		},
+		{
+			"multiple status'",
+			[]models.RunStatus{models.RunStatusPendingBridge, models.RunStatusPendingConfirmations},
```
Comment (se3000) [inline store/models/orm_test.go]: 👍

File: web/integration_test.go [hunk 2]
```diff
@@ -284,10 +284,10 @@ func TestIntegration_ExternalAdapter_Pending(t *testing.T) {
 	cltest.CreateBridgeTypeViaWeb(t, app, bridgeJSON)
 	j = cltest.FixtureCreateJobViaWeb(t, app, "../internal/fixtures/web/random_number_bridge_type_job.json")
 	jr := cltest.CreateJobRunViaWeb(t, app, j)
-	jr = cltest.WaitForJobRunToPend(t, app.Store, jr)
+	jr = cltest.WaitForJobRunToPendExternal(t, app.Store, jr)
```
Comment (se3000) [inline web/integration_test.go]: Shouldn't this be `cltest.WaitForJobRunToPendBridge`?

File: store/models/common.go [hunk 3]
```diff
@@ -21,21 +21,21 @@ const (
 	RunStatusInProgress = RunStatus("in progress")
 	// RunStatusPendingConfirmations is used for when a run is awaiting for block confirmations.
 	RunStatusPendingConfirmations = RunStatus("pending_confirmations")
-	// RunStatusPendingExternal is used for when a run is waiting on the completion
+	// RunStatusPendingBridge is used for when a run is waiting on the completion
 	// of another event.
-	RunStatusPendingExternal = RunStatus("pending_external")
+	RunStatusPendingBridge = RunStatus("pending_bridge")
```
Comment (se3000) [inline store/models/common.go]: Definitely prefer these names, as the only part of the system that should know about anything "external" are the bridges.

File: store/models/common.go [hunk 4]
```diff
@@ -21,21 +21,21 @@ const (
 	RunStatusInProgress = RunStatus("in progress")
```
Comment (se3000) [inline store/models/common.go]: I also prefer the underscored status value. Let's add an underscore here for consistency?

File: services/ethereum_listener.go [hunk 5]
```diff
@@ -77,12 +77,12 @@ func (el *EthereumListener) Disconnect() {
 
 // OnNewHead resumes all pending job runs based on the new head activity.
 func (el *EthereumListener) OnNewHead(head *models.BlockHeader) {
-	pendingRuns, err := el.Store.PendingJobRuns()
+	pendingRuns, err := el.Store.JobRunsWithStatus(models.RunStatusPendingConfirmations)
```
Comment (se3000) [inline services/ethereum_listener.go]: 🔥 🔥 🔥

File: store/models/common.go (hunk 4 above)
Comment (rupurt) [reply store/models/common.go]: Added in #214

File: web/integration_test.go (hunk 2 above)
Comment (rupurt) [reply web/integration_test.go]: Amended commit and force pushed

**Pull request merged**